m4/check_one_thread_per_ltsf.m4

```diff
@@ -0,0 +1,17 @@
+dnl Check for debug build
```

Comment:
Changes are needed in the Configuration.cpp as well to ensure number of worker threads equal number of schedule queues. Currently you can set these parameters separately and they hold default values which are not 1:1.

Reply:
In the Configuration.hpp and Configuration.cpp place the #if !defined(ONE_THREAD_PER_LTSF) on the number of worker threads code. To summarize, deactivate the run time flag for number of worker threads and just fill that variable's value with the number of LTSF queues.

Reply:
I added the code, it was only necessary to modify configuration.cpp . I think that the assert that I added is redundant. Should I keep it in as a safety measure?